packages/azure-themes/src/azure/styles/CommandBarButton.styles.ts

```diff
@@ -1,23 +1,32 @@
 import { ITheme } from 'office-ui-fabric-react';
 import { getFocusStyle } from 'office-ui-fabric-react/lib/Styling';
+import { IExtendedSemanticColors } from '../IExtendedSemanticColors';
 import { IButtonStyles } from 'office-ui-fabric-react/lib/Button';
 import { FontSizes } from '../AzureType';
 
 export const CommandBarButtonStyles = (theme: ITheme): Partial<IButtonStyles> => {
   const { semanticColors } = theme;
+  const extendedSemanticColors = semanticColors as IExtendedSemanticColors;
 
   return {
     icon: {
       color: semanticColors.focusBorder,
     },
+    iconChecked: {
+      color: '#0078D4 !important', //semanticColors.focusBorder
```

Comment:
is this comment necessary?
either remove the comment or replace it with more descriptive words.